data/scripts/events/player/default_onUpdateStorage.lua

```diff
@@ -7,7 +7,7 @@ ec.onUpdateStorage = function(creature, key, value, oldValue, isSpawn)
 		return
 	end
 
-	local player = Player(creature)
+	local player = Player(creature:getId())
```

Comment:
totally unnecessary.

Reply:
I've triple checked and it doesn't work by passing the creature userdata to the player constructor... 

Can you test it by yourself, please?

Reply:
I think you're right, I tested it on my custom engine and it worked, however it didn't work on TFS master, sorry for the inconvenience, I'll approve it.